### ISSUE TITLE
Requests dependency obsolete

### ISSUE THREAD

**Issue by hex007** [wileyyugioh/kissanime_dl#8]
```
pkg_resources.DistributionNotFound: The 'requests==2.9.1' distribution was not found and is required by kissanime-dl
```

Current version of installed  requests is 2.10

**Comment by hex007** [comment]
Fixed in commit 1eaaf97958890f997856cb6e54f2b456dfbeb18a

**Comment by hex007** [comment]
Dependencies are still obsolete. Currently running new versions of packages

**Comment by wileyyugioh** [comment]
fixed in https://github.com/wileyyugioh/kissanime_dl/commit/cedfd2b0f9dcf0254b6a0267b206dbec02d50a1d

**Comment by hex007** [comment]
@wileyyugioh Requests dependency can use ">=" format as it is always running behind with every install

**Comment by wileyyugioh** [comment]
Generally I don't like to do that because pinning versions prevents code from breaking every time a library is updated but since requests is pretty stable I'll make an exception.

**Comment by wileyyugioh** [comment]
fixed in 8181ac520f31e12318f467ff7c89a1143e0631e9
